tests/cases/corner_cases.cddl

```diff
@@ -26,6 +26,12 @@ Numbers2 = [
 	big_nint2: -0x80000000,
 ]
 
+NumberMap = {
```

Comment:
Should we have a comment here explaining why its a corner case?

Reply:
Added